tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -124,7 +124,7 @@
     "## Single-Qubit Gates <a name=\"single_gates\"/>\n",
```

Comment:
<p>I think you change to pi/4 to make this gate different from below. Maybe you can change one of the pi/4 to a different values as well, for example U(pi/2, pi/4, pi/8)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Updated on latest commit. Addressed.